client/src/components/Profile/ProfileContainer.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import Profile, {IUserData} from './Profile';
+import { getUser } from '../../store/profile/actions';
+
+interface IStateToProps {
+    user: IUserData;
+    loading: boolean;
+}
+
+interface IDispatchToProps {
+    getUser: () => void;
+}
+
+interface IState {
+    profile: IStateToProps;
+}
+
+class ProfileContainer extends React.Component <any> {
+    public render(): JSX.Element {
+        return (
+            <div className='container'>
+                <Profile
```

Comment:
do we need this container? maybe it is enough to use just profile?

Reply:
Yes, I need it. Because profile loader should be in container to look good and not on a full screen width.